tiles-generation/dockerAssets/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM overv/openstreetmap-tile-server:v1.8.2
```

Comment:
Shall we make this version configurable? So that when they release new version, we don’t have to update the file every time